src/BuildScriptGenerator/DotNetCore/DefaultAspNetCoreWebAppProjectFileProvider.cs

```diff
@@ -81,15 +84,29 @@ public string GetProjectFile(ISourceRepo sourceRepo)
             // to web sdk.
             if (projectFile == null)
             {
-                var projectFiles = sourceRepo
-                    .EnumerateFiles($"*.{DotnetCoreConstants.ProjectFileExtensionName}", searchSubDirectories: true);
+                // search for .csproj files
+                var projectFiles = GetAllProjectFilesInRepo(
+                        sourceRepo,
+                        DotnetCoreConstants.CSharpProjectFileExtensionName);
```

Comment:
Can't we get both queries in a single call to reduce FS IO? For example `dir *.csproj *.fsproj` works.

Reply:
Yeah, I did try looking for it but this Directory api does not support it...however, I think `.csproj` is still the most common scenario and is always looked up first